### ISSUE TITLE
Feature Request: Add cooldown on thetime, timeleft and ff commands

### ISSUE THREAD

**Issue by Maxi605** [alliedmodders/amxmodx#557]
Basicly my request is to add a cooldown on how much you can use the thetime/timeleft/ff (Friendlyfire) commands, as it is right know you can flood them non stop.

**Comment by Arkshine** [comment]
Just remembered that there is already an anti-flood plugin. Aren't you using it?

**Comment by Maxi605** [comment]
I am, after spamming the command 3 or 4 times the anti-flood kicks in, but the chat (and audio) gets really flooded

**Comment by WPMGPRoSToTeMa** [comment]
IMO `thetime` and `timeleft` should print and pronounce time only for the sender player.

**Comment by Arkshine** [comment]
Both `thetime` and `timeleft` commands play the voice only on the player who typed the command.

**Comment by WPMGPRoSToTeMa** [comment]
@Arkshine ah, it pronounces only to the sender, but prints for all.

**Comment by Arkshine** [comment]
It might be useful to have a special cooldown to limit the message for all players. Like, if the limit is hit (meaning if a message for all players has been sent a moment ago), then we send only the message the sender.

**Comment by WPMGPRoSToTeMa** [comment]
@Arkshine I don't know why it was designed to print time for all.

**Comment by Arkshine** [comment]
Probably because it's an useful information for everyone, or can be handy depending on the situation. I'm fine with having new cvars to control such behavior.

**Comment by OciXCrom** [comment]
Agreed. I submitted a pull request to add 2 new cvars in the plugin (#558).

**Comment by Maxi605** [comment]
thetime and timeleft are very useful information for everyone, yes, but still, imagine if 10 out of 32 players start sending the same message in chat, it not only prints your line (username : thetime) but also 2 or 3 lines of the information of the command (Don't remember how many lines, i think is 2), i'm cool with having it as it is but with a cooldown to stop the flooding/abusing of the command (as Arkshine said).
